ed25519jni/rust/src/lib.rs

```diff
@@ -37,6 +40,143 @@ pub extern "system" fn Java_org_zfnd_ed25519_Ed25519Interface_getVerificationKey
     unwrap_exc_or(&env, res, ptr::null_mut())
 }
 
+// SigningKeySeed bytes -> DER bytes
+#[no_mangle]
+pub extern "system" fn Java_org_zfnd_ed25519_Ed25519Interface_getSigningKeySeedEncoded(
+    env: JNIEnv<'_>,
+    _: JClass<'_>,
+    sks_bytes: jbyteArray,
+) -> jbyteArray {
+    let res = panic::catch_unwind(|| {
+        let mut sks = [0u8; 32];
+        sks.copy_from_slice(&env.convert_byte_array(sks_bytes).unwrap());
+        let sk = SigningKey::from(sks);
+
+        Ok(env.byte_array_from_slice(sk.to_pkcs8_der_v1().unwrap().as_bytes()).unwrap())
+    });
+    unwrap_exc_or(&env, res, ptr::null_mut())
+}
+
+// SigningKeySeed bytes -> PEM string
+#[no_mangle]
+pub extern "system" fn Java_org_zfnd_ed25519_Ed25519Interface_getSigningKeySeedPEM(
```

Comment:
Subtle point that needs to be addressed (probably in the JNI README). Since the PEM feature isn't compiled by default, I believe the PEM calls will cause faults if anybody attempts to generate a JAR without adding the feature. This is worth calling out and resolving.

Reply:
Can't this be done by simply enabling the feature in `ed25519jni/rust/Cargo.toml` or am I missing something?

Reply:
Yeah, it should be possible. I'll add it and make sure nothing breaks.